pkg/R/RDD.R

```diff
@@ -1286,48 +1311,44 @@ setMethod("partitionBy",
             #  partitionFunc <- hashCode
             #}
 
-            depsBin <- getDependencies(partitionFunc)
-            depsBinArr <- .jarray(depsBin)
+            depsBinArr <- getDependencies(partitionFunc)
 
-            serializedHashFunc <- serialize(as.character(substitute(partitionFunc)),
-                                            connection = NULL,
-                                            ascii = TRUE)
-            serializedHashFuncBytes <- .jarray(serializedHashFunc)
-
-            packageNamesArr <- .jarray(serialize(.sparkREnv$.packages,
+            serializedHashFuncBytes <- serialize(as.character(substitute(partitionFunc)),
                                                  connection = NULL,
-                                                 ascii = TRUE))
```

Comment:
Same as before. Lets investigate this is https://sparkr.atlassian.net/browse/SPARKR-179